index.html

```diff
@@ -6,15 +6,181 @@
+            </h3>
+          </div>
+          <div class="col-sm-6 col-lg-12">
+            <p class="text-left font-coco fs-3 fw-bold lh-1">
+              2021.10.15(Tue) ~ 16(Thu)
+            </p>
+            <h4 class="display-6 fs-5 mb-3">
+              @virtual, due to COVID-19
+            </h4>
+          </div>
+        </div>
+      </div>
+    </div>
+  </section>
+
+  <section class="program bg-dark bg-pattern py-5">
+    <div class="wrapper container-sm">
+      <h2 class="heading text-white text-center mb-5">Main Program</h2>
+      <div class="cards d-flex flex-column flex-md-row gap-3">
+        <div class="card bg-semi-trans d-flex flex-row flex-md-column gap-3 align-items-center p-3">
+          <img class="w-30" src="images/icons/lecture.svg" alt="">
+          <h3 class="text-bold accent mb-0">Lecture</h3>
+          <p class="text-light mb-0">Lorem ipsum dolor sit amet consectetur adipisicing elit. Nesciunt incidunt debitis.
+          </p>
+        </div>
+        <div class="card bg-semi-trans d-flex flex-row flex-md-column gap-3 align-items-center p-3">
+          <img class="w-30" src="images/icons/lecture.svg" alt="">
+          <h3 class="text-bold accent mb-0">Lecture</h3>
+          <p class="text-light mb-0">Lorem ipsum dolor sit amet consectetur adipisicing elit. Nesciunt incidunt debitis.
+          </p>
+        </div>
+        <div class="card bg-semi-trans d-flex flex-row flex-md-column gap-3 align-items-center p-3">
+          <img class="w-30" src="images/icons/lecture.svg" alt="">
+          <h3 class="text-bold accent mb-0">Lecture</h3>
+          <p class="text-light mb-0">Lorem ipsum dolor sit amet consectetur adipisicing elit. Nesciunt incidunt debitis.
+          </p>
+        </div>
+        <div class="card bg-semi-trans d-flex flex-row flex-md-column gap-3 align-items-center p-3">
+          <img class="w-30" src="images/icons/lecture.svg" alt="">
+          <h3 class="text-bold accent mb-0">Lecture</h3>
+          <p class="text-light mb-0">Lorem ipsum dolor sit amet consectetur adipisicing elit. Nesciunt incidunt debitis.
+          </p>
+        </div>
+        <div class="card bg-semi-trans d-flex flex-row flex-md-column gap-3 align-items-center p-3">
+          <img class="w-30" src="images/icons/lecture.svg" alt="">
+          <h3 class="text-bold accent mb-0">Lecture</h3>
+          <p class="text-light mb-0">Lorem ipsum dolor sit amet consectetur adipisicing elit. Nesciunt incidunt debitis.
+          </p>
+        </div>
+      </div>
+      <div class="footer d-flex mt-5">
+        <button type="button" class="btn btn-custom text-light mx-auto p-3 bg-accent">Join Conference 2021</button>
+        <a class="d-none text-center" href="#">Learn More Information</a>
+      </div>
+    </div>
+  </section>
```

Comment:
- [ ] In mobile view, the icons of this section are too big. Please try and reduce their sizes.